src/actions/constants.js

```diff
@@ -67,13 +67,13 @@ export const INCREMENT_UNSEEN_THREADS = 'INCREMENT_UNSEEN_THREADS'
 export const JOIN_COMMUNITY_WITH_CODE = 'JOIN_COMMUNITY_WITH_CODE'
 export const LEAVE_COMMUNITY = 'LEAVE_COMMUNITY'
 export const LEAVE_COMMUNITY_PENDING = LEAVE_COMMUNITY + _PENDING
+export const LOCATION_CHANGE = '@@router/LOCATION_CHANGE' // this is the react-router action type
```

Comment:
Just did poked around and this looks like the more idiomatic way to do this:
`import { LOCATION_CHANGE } from 'react-router-redux'`
So do that below when it's used and remove this pass through here.